config/locales/signal/unknown_content.txt

```diff
@@ -0,0 +1 @@
+Unsere Software kann derzeit Textnachrichten verarbeiten, aber noch nicht andere Inhalte wie Bilder, Sprachnachrichten oder Dateianhänge. In der Zwischenzeit senden Sie bitte Nachrichten mit solchen Inhalten an Astrid Csuraji, Telefonnummer <ASTRIDS_TELEFONNUMMER>. Herzlichen Dank!
```

Comment:
@FrauCsu can I add your real phone number here?

Reply:
No, please don't

Reply:
Can't we use the prepaid number @drjakob bought?

Reply:
Another option (which I think I would prefer) is just leaving a placeholder. The signal unknown content text is editable in the settings and every project can customize it with contact data they prefer. Additionally, if we put in an all_caps placeholder like <KONTAKTDATEN HINZUFÜGEN> people might even be more aware of having to adapt this part when setting up their own project.